.github/workflows/release.yml

```diff
@@ -129,7 +131,22 @@ jobs:
         with:
           name: azureauth-${{ github.event.inputs.version }}-${{ matrix.runtime }}
           path: azureauth-${{ github.event.inputs.version }}-${{ matrix.runtime }}
-
+  
+  # Build and sign Linux binaries on Azure DevOps and publish them to GitHub and packages.microsoft.com.
+  linux_release:

```

Comment:
I'm trying to come up with better name for this job. Looking for suggestions :)

Reply:
Better name for `linus_release` ?

Reply:
I disagree with `linus_release`

Reply:
I'm good with `linux_release`.

Reply:
You could use `linux_build_and_sign` to be more specific.